### ISSUE TITLE
"TTM_ADDTOOL failed" panic when running under Wine on Linux

### ISSUE THREAD

**Issue by nadnam** [lxn/walk#399]
Program is compiled with resources included and works under Windows correctly.
Compile platform is Debian and it is cross compiled to windows.386. Both CGO and no CGO options are tested, same result.

"Solution" is found in line:
 https://github.com/lxn/walk/blob/02935bac0ab8448d5f9bf72ebeeb7ca0d5553f9b/tooltip.go#L190

When ti.CbSize is set to 42 then program runs correctly with tooltips on both windows and linux under wine.
If value is set to 0 to ~40 range, wine works with tooltips but on windows tooltips are not shown.
If value is 48 (as is result of uint32(unsafe.Sizeof(ti)) on my computer) program crashes on start under wine, but works correctly on windows.

So, 42, answer to everything?

**Comment by eurake** [comment]
on windows 10  happen

**Comment by jicg** [comment]
on windows 7 happen

**Comment by logrusorgru** [comment]
Solved for wine64, lol.
```go
// if win.FALSE == tt.SendMessage(win.TTM_ADDTOOL, 0, uintptr(unsafe.Pointer(&ti))) {
// 	return newError("TTM_ADDTOOL failed")
// }
```

**Comment by majimboo** [comment]
Having the same issue when running `go run *.go`

**Comment by kjk** [comment]
`TTM_ADDTOOL failed` is usually caused by not embedding a manifest as a resource in the executable.

For more information read "Compiling as a windows application" in https://www.programming-books.io/essential/go/e87631a7f8a94e8f92f9114e8e6baf0a-a-first-window-app

**Comment by aymanhs** [comment]
You need to add the manifest to the folder now.  This will solve the issue.

See Issue #444

**Comment by iwind** [comment]
> You need to add the manifest to the folder now. This will solve the issue.
> 
> See Issue #444

Yes, you can run exe with the manifest, but you can not `go run main.go` even with manifest, it make so difficult to debug for us.
